src/styles/index.scss

```diff
@@ -1,3 +1,82 @@
-body {
+$main-accent: #060b35;
+$secondary: #616070;
+$blue_accent: #00acdc;
+
+@import "blocks/card";
+@import "blocks/stars";
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+html {
+  background-color: darkgray;
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+.block__reviews {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  font-size: 10px;
+  line-height: 14px;
+  color: $main-accent;
+  text-align: right;
+}
+
+.price {
+  display: flex;
+  width: 166px;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 16px;
+
+  &__text {
+    color: $secondary;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__number {
+    text-align: right;
```

Comment:
remove text-align: right; and pay attention to styles you've add as you have a lot of redundant styles